src/features/program/api/useGetStrapiProgramOptions.ts

```diff
@@ -0,0 +1,165 @@
+import { useQuery, UseQueryOptions } from '@tanstack/react-query'
+import { useApiClient } from '~/providers/ApiClientProvider'
+import {
+  ProgramBranchResponse,
+  ProgramCategoryResponse,
+  ProgramLanguageResponse,
+  ProgramMethodResponse,
+  ProgramTopicResponse
+} from '../types/strapiProgramOption'
+
+export const useGetStrapiProgramCategories = (
+  programCategoryParams: { title?: string },
+  options?: UseQueryOptions<
+    unknown,
+    unknown,
+    ProgramCategoryResponse[],
+    (string | undefined)[]
+  >
+) => {
+  const { title } = programCategoryParams
+
+  const { axiosStrapi, api } = useApiClient()
+
+  return useQuery(
+    ['program-categories', title],
```

Comment:
```suggestion
    ['strapi-program-categories', title],
```
buat yang get dari strapi keynya kaya gini aja biar kalau ada yang get dari BE purwadhika ga bentrok key nya